hubspot3/tickets.py

```diff
@@ -1,9 +1,12 @@
 """
 hubspot tickets api
 """
+import itertools
+
 from hubspot3.base import BaseClient
-from hubspot3.utils import get_log
-from typing import Dict
+from hubspot3.utils import prettify, get_log
+
+from typing import Dict, Union, List
```

Comment:
`typing` is part of the standard library and should be after the `itertools` import. The correct  import order is `Dict, List, Union`, following PEP8.